### ISSUE TITLE
Redesign default value method for structs

### ISSUE THREAD

**Issue by mdwhitten** [NISystemsEngineering/rfic-reference-design-libraries-csharp#29]
In pull request #26 @smooresni suggested a better syntax for default values:
```c#
public struct InstrumentConfiguration
        {
            public string ReferenceClockSource;
            //etc.
            public static InstrumentConfiguration GetDefault()
            {
                return new InstrumentConfiguration()
                {
                    ReferenceClockSource = RfsgFrequencyReferenceSource.PxiClock.ToString(),
                    //etc.
                };
            }
        }
```

This allows for the following simple syntax to utilize: 
```c#
InstrumentConfiguration myConfig = InstrumentConfiguration.GetDefault();
```
This is a much cleaner solution.

**Comment by smooresni** [comment]
@mdwhitten will submit pull request for this enhancement

**Comment by smooresni** [comment]
see #35

**Comment by mdwhitten** [comment]
Successfully implemented in #35
